pkg/controller/webhooks/common.go

```diff
@@ -0,0 +1,17 @@
+package webhooks
```

Comment:
Maybe call the file `claim_deletionprotection.go` or something. Common sounds like it's some sort of library for various common stuff.

Reply:
Done